pkg/taskutil/spec.go

```diff
@@ -119,6 +119,16 @@ func MetadataSpecURL() (string, error) {
 	return "", nil
 }
 
+type DefaultJSONSpecDecoder struct{}
+
+func (u DefaultJSONSpecDecoder) DecodeSpec(r io.Reader, v interface{}) error {
```

Comment:
Is this used anywhere? Seems like it isn't needed as it just directly wraps the JSON library?

Reply:
Ah heh, this was leftover from trying to add my own before I found the upstream change.